tests/list.c4m

```diff
@@ -25,11 +53,15 @@ for item in x[1:-1] {
   sum += item
 }
 
-assert sum == 80
-x[1:-1] = [5]
+print(sum)
+
+x[1:-1] = [100]
+print(x)
```

Comment:
Well, it was intended, but because my memory of Python semantics were off. I just went and looked, to see that I have to omit the colon for 'to the end of the list'.  For now, this is right, and I'll revisit it soon.